### ISSUE TITLE
Add soundtrack from the original game.

### ISSUE THREAD

**Issue by MikillRosen** [MustaphaTR/sole-survivor#10]
Also, replace the main menu theme with the one from the original game.

**Comment by MikillRosen** [comment]
actually, you already did. But didn't replace the main menu theme. Sorry for my ignorance.

**Comment by MustaphaTR** [comment]
Yes, mod already supports original SS music, but i'm not sure how the game would act if i tried to play it when player doesn't have SS music but has TD music and i didn't want to remove the ability to play the TD music. The OpenRA music player needs some improvements i guess. It could be helpful if we could categorise the music and disable stuff we don't want.
